onadata/apps/logger/models/instance.py

```diff
@@ -99,7 +99,7 @@ def update_xform_submission_count(instance, created, **kwargs):
         )
 
 
-def nullify_exports_time_of_last_submission(sender, instance, **kwargs):
+def nullify_exports_time_of_last_submission(instance, **kwargs):
```

Comment:
Yes, let's revert it per our discussion because Django says the receivers always need to accept `sender`, and there's no guarantee that `sender` will always be passed as a keyword argument instead of positionally